src/service/standard-actions-impl.js

```diff
@@ -267,6 +269,35 @@ export class StandardActions {
       return this.handleHide(invocation);
     }
   }
+
+  /**
+   * Handles "toggleClass" action.
+   * @param {!./action-impl.ActionInvocation} invocation
+   * @return {?Promise}
+   */
+  handleToggleClass(invocation) {
+    if (!invocation.satisfiesTrust(ActionTrust.HIGH)) {
+      return null;
+    }
+
+    const target = dev().assertElement(invocation.node);
```

Comment:
This should have also been a user assertion.

Reply:
I don't there is a way for `node` to be null or not an element through normal user-facing `on` attribute. If this ever happens either action service has a bug or some internal code is dispatching an invocation.